core/app/modules/kit-library/data/endpoints/kits-download-link.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace Elementor\Core\App\Modules\KitLibrary\Data\Endpoints;
+
+use Elementor\Data\Base\Endpoint;
+use Elementor\Core\App\Modules\KitLibrary\Data\Repository;
+use Elementor\Core\App\Modules\KitLibrary\Data\Exceptions\Wp_Error_Exception;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Kits_Download_Link extends Endpoint {
+	public function get_name() {
+		return 'download-link';
+	}
+
+	protected function register() {
+		$this->register_route(
+			'/(?P<id>[\w-]+)/',
+			\WP_REST_Server::READABLE,
+			function ( $request ) {
+				return $this->base_callback( \WP_REST_Server::READABLE, $request, false );
+			},
+			[
+				'id' => [
+					'description' => 'Unique identifier for the object.',
+					'type' => 'string',
+					'required' => true,
+				],
+			]
+		);
+	}
+
+	public function get_item( $id, $request ) {
+		/** @var Repository $repository */
+		$repository = $this->controller->repository;
+
+		try {
+			$data = $repository->get_download_link( $id );
+		} catch ( Wp_Error_Exception $exception ) {
+			return new \WP_Error( $exception->getCode(), $exception->getMessage(), [ 'status' => $exception->getCode() ] );
+		} catch ( \Exception $exception ) {
+			return new \WP_Error( 'server_error', __( 'Something went wrong.', 'elementor' ) );
+		}
+
+		return new \WP_REST_Response( [
+			'data' => $data,
+		] );
```

Comment:
Why not?
```suggestion
		return [
			'data' => $data,
		];
```

Reply:
I just like that the return value of the method is explicit